<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="logBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>941</width>
     <height>231</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="funcRun">
      <property name="text">
       <string>Func Run</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="tranBtn">
      <property name="text">
       <string>Transaction</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="reqCode">
      <property name="text">
       <string>Request code</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="reqInfo">
      <property name="text">
       <string>Request Info</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>601</width>
     <height>82</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label1">
          <property name="text">
           <string>input argu :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="argu_1"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>function :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="function_name"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>hello()</slot>
 </slots>
</ui>
